testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -528,6 +507,51 @@ QUnit.module('datebox tests', moduleConfig, () => {
     });
 });
 
+QUnit.module('toolbar buttons testing', {}, () => {
+    const buttons = [
+        { key: 'todayButtonText', name: 'Today', text: 'newTodayText', selector: TODAY_BUTTON_SELECTOR },
+        { key: 'applyButtonText', name: 'Done', text: 'newDoneText', selector: APPLY_BUTTON_SELECTOR },
+        { key: 'cancelButtonText', name: 'Cancel', text: 'newCancelText', selector: CANCEL_BUTTON_SELECTOR },
+    ];
+
+    buttons.forEach(button => {
+        QUnit.test(`Customize "${button.name}" button on init`, function(assert) {
+            const $dateBox = $('#dateBox').dxDateBox({
+                type: 'datetime',
+                pickerType: 'calendarWithTime',
```

Comment:
There is no such a pickerType
```suggestion
                pickerType: 'calendar',
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/22991/commits/6db8ca42fc7d248ad8eb6711a8299e46aae5347b